### ISSUE TITLE
Output magnetic variation with NMEA RMC sentences

### ISSUE THREAD

**Issue by johnboiles** [nohal/wmm_pi#6]
Right now, when a route is activated in OpenCPN, three NMEA sentences get sent out any available output serial port: RMC, RMB & APB. RMC messages have a field for magnetic variation, but they are currently blank. It'd be awesome to populate that field based on data from the WMM plugin.

@nohal if this makes sense to you, I'm happy to scope this out.

**Comment by nohal** [comment]
John...
Are you sure? Looking at routeman.cpp:600 and onwards, I have an impression that the variation should get populated.

Pavel

**Comment by johnboiles** [comment]
I haven't dug into the code yet, just reporting what I've seen experimentally. I'll take a look again, last time I tried, this is the sentence I got as output:

```
$ECRMC,064535,A,3751.245,N,12218.346,W,0.055,nan,231114,,,*7C
```

I was on a 3.3 beta but don't remember which one. Let me try again with a more recent build and I'll report back.

**Comment by johnboiles** [comment]
Ah yeah I'm seeing it! Possible I was running an old build when I logged a bunch of that NMEA data, or I was running a debug build w/o the plugin installed. At any rate, my bad. Thanks for the prompt response!
